tutorials/ai-core-and-LP-provisioning-GA/ai-core-and-LP-provisioning-GA.md

```diff
@@ -0,0 +1,118 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>license, tutorial>beginner, topic>artificial-intelligence, topic>provisioning, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+---
+
+# Use Boosters for Free Tier Use of SAP AI Core and SAP AI Launchpad
+
+## Prerequisites
+- A BTP global account
+
+## You will learn
+- How to access your SAP BTP global account
+- How to use boosters to get started with SAP AI Core and SAP AI Launchpad
+
+## About SAP AI Core and SAP AI Launchpad
```

Comment:
If this is introductory information, then the text after the two hashtags should be "Intro."  After that you can add your information.  For the subsequent sections, use four hashtags to offset the headers.